cli/tests/activate.bats

```diff
@@ -3336,9 +3336,12 @@ EOF
 
 # bats test_tags=activate,activate:attach
 @test "attach doesn't break MANPATH" {
-  # We don't need an environment, but we do need wait_for_watchdogs to have a
-  # PROJECT_DIR to look for
-  project_setup_common
+  project_setup
+
+  # Ensure that an empty MANPATH is replaced with something with a trailing colon.
```

Comment:
TIL from `man(1)`:

> If MANPATH begins with a colon, it is appended to the default list; if it ends with a colon, it is prepended to the default list; or if it contains two adjacent colons, the search path is inserted between the colons.  If none of these conditions are met, it overrides the standard search path.

It would be worth noting why this is important:
```suggestion
  # Ensure that an empty MANPATH is replaced with something with a trailing
  # colon so that the default list is honoured as a fallback.
```

Reply:
Sounds good, this is commented extensively in the code as well. Will commit your suggestion.